tests/unit/parser/test_s3.py

```diff
@@ -105,3 +110,33 @@ def test_s3_eventbridge_notification_object_restore_completed_event():
     assert model.detail.requester == raw_event["detail"]["requester"]
     assert model.detail.restore_expiry_time == raw_event["detail"]["restore-expiry-time"]
     assert model.detail.source_storage_class == raw_event["detail"]["source-storage-class"]
+
+
+def test_s3_sqs_event_notification():
+    raw_event = load_event("sqsS3Event.json")
+    model = S3SqsEventNotificationModel(**raw_event)
+
+    body = json.loads(raw_event["Records"][0]["body"])
+
+    assert model.Records[0].body.Records[0].eventVersion == body["Records"][0]["eventVersion"]
+    assert model.Records[0].body.Records[0].eventSource == body["Records"][0]["eventSource"]
+    assert model.Records[0].body.Records[0].eventTime == datetime.fromisoformat(
+        body["Records"][0]["eventTime"].replace("Z", "+00:00")
+    )
+    assert model.Records[0].body.Records[0].eventName == body["Records"][0]["eventName"]
+
+
+def test_s3_sqs_event_notification_body_invalid_json():
+    raw_event = load_event("s3Event.json")
```

Comment:
Change this line too

Reply:
This test was wrong. I was using a S3 JSON file to simulate an invalid body, but it is supposed to use an invalid json body (like a string).

Fixed.

Reply:
If you compare against my original code in commit 251d8ba, I used `sqsEvent.json`, which is another way to solve this problem.

This is also why I originally named the classes `SqsS3...` (because the SQS payload literally wraps the S3 payload) rather than `S3Sqs...` (the direction of traffic). They're simply different ways of conceptualising the same scenario, neither is particularly right or wrong I guess!

Reply:
You're right @theipster, I used the wrong words to explain why I changed the test. It's not a question of being right or wrong, it's just different ways of solving the same question. 

Thank you very much for this feedback, it will help me to avoid mistakes like this in the future.